### PR TITLE
WIP: DO NOT MERGE: Apache Arrow 2022-10 board report

### PR DESCRIPTION
**Pull request by kou**
This is a draft of Apache Arrow 2022-10 board report. Committers can push directly to this branch. Others can leave suggestions.

This is never merged to apache/arrow. This will be reported by https://reporter.apache.org/wizard/?arrow .

### CODE REVIEW

File: board-report-2022-10.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- 7.0.1 was released on 2022-07-19.
+- 8.0.1 was released on 2022-07-19.
+- RS-DF-10.0.0 was released on 2022-07-15.
+- RS-18.0.0 was released on 2022-07-11.
+
+## Community Health:
+
+- We're voting a new Apache Arrow Database Connectivity (ADBC)
+  interface developing at https://github.com/apache/arrow-adbc .
+- We'll deprecate unmaitained Apache Arrow Plasma object store.
```
Comment (cyb70289) [inline board-report-2022-10.md:47]: typo `unmaintained`

**Review by lidavidm — APPROVED** [review]

File: board-report-2022-10.md (hunk 1 above)
Comment (kou) [reply board-report-2022-10.md]: Good catch!

File: board-report-2022-10.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+## Description:
+
+The mission of Apache Arrow is the creation and maintenance of
+software related to columnar in-memory processing and data interchange
```
[the remark this answers is not in the document]
Comment (kou) [reply board-report-2022-10.md]: Thanks!

**Review by cyb70289 — APPROVED** [review]

**Comment by kou** [comment]
Submitted. Thanks!

**Review by alamb — COMMENTED** [review]
Thanks @kou

**Pull request closed without merging**